cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/connector/resource/AwsCloudWatchService.java

```diff
@@ -0,0 +1,85 @@
+package com.sequenceiq.cloudbreak.cloud.aws.connector.resource;
+
+import static com.sequenceiq.cloudbreak.cloud.model.CloudStack.CLOUDWATCH_CREATE_PARAMETER;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+import com.amazonaws.services.cloudwatch.AmazonCloudWatchClient;
+import com.amazonaws.services.cloudwatch.model.AmazonCloudWatchException;
+import com.amazonaws.services.cloudwatch.model.DeleteAlarmsRequest;
+import com.amazonaws.services.cloudwatch.model.Dimension;
+import com.amazonaws.services.cloudwatch.model.PutMetricAlarmRequest;
+import com.sequenceiq.cloudbreak.cloud.aws.AwsClient;
+import com.sequenceiq.cloudbreak.cloud.aws.view.AwsCredentialView;
+import com.sequenceiq.cloudbreak.cloud.model.CloudInstance;
+import com.sequenceiq.cloudbreak.cloud.model.CloudResource;
+import com.sequenceiq.cloudbreak.cloud.model.CloudStack;
+
+@Service
+public class AwsCloudWatchService {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(AwsCloudWatchService.class);
+
+    private  static final String ALARM_SUFFIX = "-Status-Check-Failed-System";
+
+    private static final int CLOUDWATCH_PERIOD = 60;
+
+    private static final int CLOUDWATCH_EVALUATION_PERIODS = 2;
+
+    private static final double CLOUDWATCH_THRESHOLD = 1.0;
```

Comment:
Could you move these to `@Values` so we can easily override them?

Reply:
done